src/internal/packager/helm/chart.go

```diff
@@ -22,6 +22,9 @@ import (
 	"helm.sh/helm/v3/pkg/storage/driver"
 )
 
+// Set the default helm client timeout to 15 minutes
+const defaultClientTimeout = 15 * time.Minute
```

Comment:
Potentially?  I pulled it to a constant to make that easier in the future, but wasn't sure of a good pattern to introduce it yet.  Most of these kinds of things have been configured in the Zarf.yaml (like no wait) so maybe that would be better?  Looking to @jeff-mccoy for any thoughts as well.

Reply:
Yeah mixed feelings on this one, we _could_ add it to the yaml spec, but I don't want to expose too much complexity. It will do a retry if it fails and 15 minutes seems like a pretty long time to reconcile, but I'm really not sure tbh. Maybe we just wait to see if we ever hit that very unusual edge case and decide then?

Reply:
Yeah I am fine with that for now, especially with the better state management during removal.  It should make resuming after failure easier too.